7.3/Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN apt-get update && apt-get install -y \
   libxpm-dev \
   libxml2-dev \
   libxslt-dev \
+  librabbitmq-dev \
```

Comment:
this was missing, and theerfore the error message "pecl amqp..... error" occured